### PR TITLE
Modernize Meson usage

### PR DESCRIPTION
**Pull request by TingPing**

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
Awesome thanks for the great work ! :+1:  Just wonder as the pot file doesn't exists anymore, how anyone can translate the extension ? 
+ The POTFILES doesn't contain all the files that have a trasnlatable string on theme, there's also the nemo-git/git.py.in

**Comment by TingPing** [comment]
> Awesome thanks for the great work ! :+1: Just wonder as the pot file doesn't exists anymore, how anyone can translate the extension ?

You can generate the file with `ninja nautilus-git-pot`, also `ninja nautilus-git-update-po`.

> The POTFILES doesn't contain all the files that have a trasnlatable string on theme, there's also the nemo-git/git.py.in

Thanks, will add.

File: README.md [hunk 1]
```diff
@@ -107,23 +97,9 @@ nautilus -q
 1- Download the uninstallation file
```
Comment (bilelmoussaoui) [inline README.md]: Can you remove that please? as  there's nothing to download anymore

**Review by bilelmoussaoui — CHANGES_REQUESTED** [review]

**Comment by bilelmoussaoui** [comment]
I've merged the PR, everything was fine. I was just too tired and i was looking at the other old version of POTFILES. Thanks again for this great PR :+1:

**Pull request merged**